jobs/config.json

```diff
@@ -10825,14 +10825,14 @@
       "--cluster=",
       "--deployment=kubernetes-anywhere",
       "--env-file=jobs/platform/gce.env",
-      "--extract=local",
-      "--gcp-project=k8s-jkns-pr-kubeadm",
       "--gcp-zone=us-central1-f",
+      "--ginkgo-parallel=30",
       "--kubeadm=pull",
-      "--kubernetes-anywhere-kubernetes-version=latest",
+      "--kubernetes-anywhere-kubernetes-version=ci/latest",
```

Comment:
Right now, we can't test the PR's control plane, but have to use the CI's latest control plane images.
Should be fixed in the future, but right now we don't push images from the bazel job, soo...